test/testcafe/spec/AuthenticatorIdPView_spec.js

```diff
@@ -75,13 +75,9 @@ const verifyErrorCustomLogoMock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/introspect')
   .respond(xhrVerifyIdpAuthentiatorErrorCustomLogo);
 
-async function setup(t, {isVerify, expectAutoRedirect} = {}, widgetOptions = undefined) {
-  const options = widgetOptions ? { render: false } : {};
+async function setup(t, {isVerify, expectAutoRedirect} = {}) {
   const pageObject = new IdPAuthenticatorPageObject(t);
-  await pageObject.navigateToPage(options);
-  if (widgetOptions) {
-    await renderWidget(widgetOptions);
```

Comment:
There is sometimes flakiness associated with waiting for the widget re-render, so instead we use `clientScripts` to override the widget options in the template before the widget bootstrap